packages/web-client/app/components/card-pay/deposit-workflow/transaction-setup/token-option/index.css

```diff
@@ -0,0 +1,82 @@
+.token-option {
+  display: grid;
+  grid-template-columns: auto 1fr 1fr;
+  align-items: center;
+  gap: var(--boxel-sp);
+  padding: var(--boxel-sp);
+  border-radius: var(--boxel-border-radius);
+  box-shadow: 0 0 0 1px var(--boxel-light-400);
+  transition: box-shadow var(--boxel-transition);
+}
+
+.token-option:hover {
+  box-shadow: 0 0 0 1px var(--boxel-dark);
+}
+
+.token-option--checked,
+.token-option:focus,
+.token-option:focus-within {
+  box-shadow: 0 0 0 2px var(--boxel-highlight);
+  outline-color: transparent;
+  outline-width: 0px;
```

Comment:
We can leave the outline-width as is for Windows High Contrast Mode, which ignores box-shadow.

https://sarahmhigley.com/writing/whcm-quick-tips/